{"name": "test_sub[sun-other]", "status": "failed", "statusDetails": {"message": "AssertionError: assert None == 'qq'\n  +None\n  -'qq'", "trace": "self = <test_pytest_HT.test_calc_new.TestCalc object at 0x000001BF0F909518>\nget_calc = <test_pytest_HT.calculator.Calculator object at 0x000001BF0F87F1D0>, get_sub_datas = ['qq', 'qq', 'qq']\n\n    @pytest.mark.run(order=2)\n    def test_sub(self, get_calc, get_sub_datas):\n        result = None\n        try:\n            result = get_calc.sub(get_sub_datas[0], get_sub_datas[1])\n            if isinstance(result, float):\n                result = round(result, 2)\n        except Exception as e:\n            print(e)\n>       assert result == get_sub_datas[2]\nE       AssertionError: assert None == 'qq'\nE         +None\nE         -'qq'\n\ntest_calc_new.py:50: AssertionError"}, "parameters": [{"name": "get_sub_datas", "value": "['qq', 'qq', 'qq']"}], "start": 1610373753615, "stop": 1610373753615, "uuid": "df382031-2bdf-4355-b104-8965812e3dca", "historyId": "ddbd3df45a7bfdc9d520282924df80c8", "testCaseId": "a1506a2b3b61cae69056fcb454df3b1b", "fullName": "test_calc_new.TestCalc#test_sub", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-B1515VS"}, {"name": "thread", "value": "3300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}